providers/hns/hns_roce_u.h

```diff
@@ -124,6 +126,9 @@ struct hns_roce_buf {
 #define BIT_CNT_PER_BYTE       8
 #define BIT_CNT_PER_LONG       (BIT_CNT_PER_BYTE * sizeof(unsigned long))
 
+#define GENMASK(h, l) \
```

Comment:
EFA already has this define.
Please reuse it.

Reply:
OK, will put it in ccan/. Thank you.

Reply:
I don't find GENMASK in CCAN, so I will achieve it in util.h.